libs/sdk-ui-kit/src/ChartSorting/AttributeDropdown/AttributeDropdown.tsx

```diff
@@ -80,18 +104,67 @@ const getSortTypeItems = (available: IAvailableSortsGroup, intl: IntlShape): ISo
             },
         );
     }
-
+    if (available.attributeSort.normalSortEnabled && bucketItem.type === "genericDate") {
```

Comment:
would not be more readable to keep all variants of normal sorting together?
if (available.attributeSort.normalSortEnabled)
{
 if(attribute) push
 if(chrono) push
 if(generic) push
}

Reply:
I had it like that originally but I needed to be able to have `Default` option as the last one below numerical sort as in the mockups and this was the easiest way.

<img width="386" alt="image" src="https://user-images.githubusercontent.com/1880148/157198528-7fd4d88f-5044-4dd0-93fc-d8745cffac2f.png">

Reply:
ok